litmus-portal/frontend/src/views/MyHub/MyHubCharts/index.tsx

```diff
@@ -0,0 +1,155 @@
+import {
+  Backdrop,
+  Card,
+  CardContent,
+  Link,
+  Typography,
+} from '@material-ui/core';
+import React, { useEffect, useState } from 'react';
+import { useQuery } from '@apollo/client';
+import { useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+import Scaffold from '../../../containers/layouts/Scaffold';
+import useStyles from './styles';
+import { GET_CHARTS_DATA } from '../../../graphql';
+import { RootState } from '../../../redux/reducers';
+import { Chart, Charts } from '../../../models/redux/myhub';
+import Loader from '../../../components/Loader';
+import Center from '../../../containers/layouts/Center';
+import HeaderSection from './headerSection';
+
+interface ChartName {
+  ChaosName: string;
+  ExperimentName: string;
+}
+
+const MyHub = () => {
+  const classes = useStyles();
+
+  const hubData = useSelector((state: RootState) => state.hubDetails);
+
+  const { HubName, RepoName, RepoURL, RepoBranch, UserName } = hubData;
+
+  const experimentDefaultImagePath = `${RepoURL.split('/')[0]}//${
+    RepoURL.split('/')[2]
+  }/${RepoURL.split('/')[3]}/${RepoName}/raw/${RepoBranch}/charts/`;
+
+  const { t } = useTranslation();
+
+  const { data, loading } = useQuery<Charts>(GET_CHARTS_DATA, {
+    variables: {
+      data: {
+        UserName,
+        RepoURL,
+        RepoBranch,
+        RepoName,
+        HubName,
+      },
+    },
+    fetchPolicy: 'cache-and-network',
+  });
+
+  const [search, setSearch] = useState('');
+
+  const changeSearch = (
+    event: React.ChangeEvent<HTMLTextAreaElement | HTMLInputElement>
+  ) => {
+    setSearch(event.target.value as string);
+  };
+
+  const [totalExp, setTotalExperiment] = useState<ChartName[]>([]);
+
+  const exp: ChartName[] = [];
```

Comment:
Remove line spacing where ever possible. It's not adding much to the readability.